RadarSDK/RadarAPIClient.m

```diff
@@ -265,30 +274,34 @@ - (void)trackWithLocation:(CLLocation *_Nonnull)location
     // remove the updatedAtMsDiff key because for replays we want to rely on the updatedAtMs key for the time instead
     [bufferParams removeObjectForKey:@"updatedAtMsDiff"];
     
-    NSMutableDictionary *requestsParams = [NSMutableDictionary new];
+    NSMutableDictionary *requestParams = [NSMutableDictionary new];
 
     BOOL replaying = options.replay == RadarTrackingOptionsReplayAll && replayCount > 0;
     
-    if (replaying) {
+    if (verified) {
+        url = [NSString stringWithFormat:@"%@/v1/track", host];
+        
+        requestParams = [params mutableCopy];
+    } else if (replaying) {
```

Comment:
Hmmm, given that the `if (verified)` clause and the `else` are the same, it would be DRYer if we could just check `replaying` first.

`replaying` is only true iff `replay: all` and there are replays waiting to be uploaded (line 279 just above), so I imagine it would be fine to do it that way.

Up to you.

Reply:
alternatively, to keep the order but make it dry, `if (verified || !replaying) ... else`

Reply:
@lmeier This simplifies the code a bit and also makes it more line-by-line consistent with Android: https://github.com/radarlabs/radar-sdk-ios/pull/215/commits/43fd39c8ccf38bbb38970157018bfd2014761898#diff-57a83f22ae98eebd6b400f6f7baecfb73b170d06629cbf8718e0a9000069d5a8R272

Reply:
Nice! I think we want to clear out `requestParams` in that if statement, as the server is expecting params in the format:
```
{ "replays": [ {...}, {...}, ...]}
```